src/ACEtk/block/FrameAndMultiplicityBlock.hpp

```diff
@@ -43,37 +46,32 @@ class FrameAndMultiplicityBlock : protected details::Base {
   /**
    *  @brief Return the number of available reactions (excluding elastic)
    */
-  unsigned int NTR() const { return this->ntr_; }
+  std::size_t NTR() const { return this->ntr_; }
 
   /**
    *  @brief Return the number of available reactions (excluding elastic)
    */
-  unsigned int numberReactions() const { return this->NTR(); }
+  std::size_t numberReactions() const { return this->NTR(); }
```

Comment:
[not directly related to this value, but design question]
Should we be casting XSS values to `int64_t` (or `int_fast64_t` if you want to be real pedantic) instead of `size_t`? I see a fair number of places in which subtraction is used to compute indices. I have yet to observe any bugs, but it is difficult to guarantee that subtraction operations on unsigned integers will behave consistently.

I know that there should never be a negative number of (say) reactions, it's just a matter of how they're used within index arithmetic.

Reply:
We'll consider this for the future.